fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/observer/FedoraEvent.java

```diff
@@ -54,8 +54,7 @@
 
     private Set<Integer> eventTypes = new HashSet<>();
     private Set<String> eventProperties = new HashSet<>();
-
-    private static final Supplier<String> pidMinter = new UUIDPathMinter();
+    private UniqueValueSupplier pidMinter = new DefaultPathMinter();
```

Comment:
Commit pushed (actually two). The nested classes had to be made static in order to use them on a static field.